src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfService.java

```diff
@@ -281,10 +281,12 @@ public List<PdfDecodedMultipartFile> convertAcasCertificatesToPdfDecodedMultipar
      * @return {@link PdfDecodedMultipartFile} with the claimant tse CYA page in pdf format.
      * @throws DocumentGenerationException if there is an error generating the PDF.
      */
-    public PdfDecodedMultipartFile convertClaimantTseIntoMultipartFile(ClaimantTse claimantTse)
+    public PdfDecodedMultipartFile convertClaimantTseIntoMultipartFile(ClaimantTse claimantTse,
+                                                                       String contactApplicationDate,
+                                                                       String contactApplicant)
         throws DocumentGenerationException {
         return new PdfDecodedMultipartFile(
-            convertClaimantTseToPdf(claimantTse),
+            convertClaimantTseToPdf(claimantTse, contactApplicationDate, contactApplicant),
```

Comment:
You can move this method to another class or create a new utility class as this class has lot's of methods which should be moved.

Reply:
done